classes/blocks/class-covers.php

```diff
@@ -389,6 +425,12 @@ private static function populate_posts_for_act_pages( &$fields ) {
 			$actions = self::filter_posts_by_ids( $fields );
 		} else {
 			$actions = self::filter_posts_for_act_pages( $fields );
+			if ( IA::is_active( IA::ACTION_POST_TYPE ) ) {
+				$actions = array_merge(
```

Comment:
From what I've tested on my local, it looks like this will result in the Action pages always showing at the beginning 🤔 I don't think that's the expected behaviour, probably we want to order all posts by publishing date?

Reply:
Oh good point, gonna fix that for this and the other PR

Reply:
Which other PR? 👀

Reply:
:smile: the [Take Action](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/802) one, same concept but you're not reviewer on that one